MaxText/goodput.py

```diff
@@ -0,0 +1,127 @@
+"""
+Goodput Prototype (TODO: Move to packaged library)
```

Comment:
Also why is this writing to GCS? Is there some metric sservice we can use?

Reply:
Writing to GCS is a lot more complicated that it seems so we'd need to think about that carefully.

Reply:
+1, I think we can readily use Cloud Logging to stream logs: https://cloud.google.com/logging/docs/write-query-log-entries-python#linux